code/__defines/subsystem-priority.dm

```diff
@@ -26,9 +25,10 @@
 #define SS_INIT_LIGHTING   	0	// Generation of lighting overlays and pre-bake. May cause openturf updates, should initialize before SSzcopy.
 #define SS_INIT_ZCOPY      -1	// Z-mimic flush. Should run after SSoverlay & SSicon_smooth so it copies the smoothed sprites.
 #define SS_INIT_XENOARCH   -2   // Xenoarch is this far below because it can infinite loop if placed in SS_INIT_MISC as it was before, due to some subsystems spawning stuff there.
-#define SS_INIT_LOBBY      -3	// Lobby timer starts here. The lobby timer won't actually start going down until the MC starts ticking, so you probably want this last
-#define SS_INIT_PING 	   -4	// Pinger for the clients
-#define SS_INIT_CHAT       -5	// To ensure chat remains smooth during init.
+#define SS_INIT_HOLOMAP    -3   // Minimap subsystem. Should be initialized after all maps, away sites, ships, planets, etc.
```

Comment:
Why does it have to be run that early? / Why cant it be run where it currently is?

Reply:
Don't you mean late? Lower and negative number means it is ran later (closer to round starting). 

Earliest it could run is probably after `SS_INIT_MISC`, but honestly I don't think it matters all that much, it just needs to run after all the mapping (planets, sites, ships, etc) stuff.